gapic-generator/lib/gapic/schema/api.rb

```diff
@@ -109,17 +110,11 @@ def protoc_options
       #   An array of the sample configuration hashes.
       def samples
         @samples ||= begin
-          sam = []
-          if protoc_options[:samples]
-            require "yaml"
-            path = protoc_options[:samples]
-            files = Dir.glob "#{path}/*.yaml"
-            files.each do |file|
-              yaml = YAML.load_file file
-              sam += yaml["samples"] if yaml["samples"]
-            end
-          end
-          sam
+          protoc_options[:samples].to_s.split(";").flat_map do |sample_path|
+            yaml = YAML.load_file sample_path
+            # TODO: Remove use of "1.2.0" magic variable
+            yaml["samples"] if yaml["schema_version"] == "1.2.0"
```

Comment:
Issue a warning to the console here as well?

Reply:
What should it say?

Reply:
It doesn't really matter. I was thinking something like: `Unexpected sample file with version X found - ignoring`. 

It probably won't matter anytime soon but just so that we're not wondering why there's no output if the version does change in the future.

Reply:
Added.

Reply:
LGTM!